src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/model/CaseData.java

```diff
@@ -354,6 +354,13 @@ public class CaseData {
     )
     private String familyCourtEmailId;
 
+    @JsonUnwrapped
```

Comment:
As per my undertanding you need to store multiple notes, not just single one ?

Reply:
I'll test if multiple Notes are being allowed or not. But yes I might have missed the multiple notes scenario. I'll test and will add